packages/block-library/src/navigation-menu/style.scss

```diff
@@ -100,10 +97,9 @@
 		&::after {
 			content: "\00a0\25BC";
 			display: inline-block;
-			font-size: 0.6rem;
+			font-size: 6px;
```

Comment:
Yep, I personally have a strong strong preference for px units over `rem` units. I'm not sure everyone does, but I also don't see the value of the `rem` unit in designs that are intrinsically pixel-based, like this connector.